32.md

```diff
@@ -157,7 +157,7 @@ considered open for public use, and not proprietary. In other words, if there is
 namespace that fits your use case, use it even if it points to someone else's domain name.
 
 Vocabularies MAY choose to fully qualify all labels within a namespace (for example,
-`["l", "com.example.vocabulary:my-label"]`. This may be preferred when defining more
+`["l", "com.example.vocabulary:my-label"]`). This may be preferred when defining more
```

Comment:
Accidental change?

Reply:
no, if you look carefully there is an unclosed parenthesis before this new one.